05/app.js

```diff
@@ -9,6 +9,39 @@ const stats = {
 
 /* tutaj umieść swój kod */
 
+const paragraphs = document.querySelectorAll('p')
+
+paragraphs.forEach(function(paragraph) {
+    paragraph.addEventListener('click', function(event) {
+        if(event.target.classList.contains('link')) {
+            event.preventDefault()
+            const linkHref  = event.target.getAttribute('href')
+
+
+            if(!stats.links.hasOwnProperty(linkHref)) {
+                stats.links[linkHref] = 0
+            }
+
+            stats.links[linkHref]++
```

Comment:
Zwróć uwagę, że ta część jest dość podobna do tej z linii 30-34 lub 37-39.
Może dałoby się stworzyć funkcję, która byłaby uruchamiana w tych przypadkach?

Będzie trzeba do takiej funkcji przekazać m. in. informacje jaka właściwość nas interesuje tj. czy `links` czy `paragraphs`